crates/lib3h/src/engine/real_engine.rs

```diff
@@ -284,24 +284,32 @@ impl<T: Transport, D: Dht, SecBuf: Buffer, Crypto: CryptoSystem> RealEngine<T, D
             Lib3hClientProtocol::FetchEntry(_msg) => {
                 // FIXME
             }
-            // HandleFetchEntryResult: Convert to DhtCommand::EntryDataResponse
+            // HandleFetchEntryResult:
+            //   - From GetAuthoringList      : Convert to DhtCommand::BroadcastEntry
+            //   - From DHT EntryDataRequested: Convert to DhtCommand::EntryDataResponse
             Lib3hClientProtocol::HandleFetchEntryResult(msg) => {
                 let maybe_space = self.get_space_or_fail(
                     &msg.space_address,
                     &msg.provider_agent_id,
-                    &format!("PublishEntry_{:?}", msg.entry.entry_address),
+                    &msg.request_id,
                     None,
                 );
                 match maybe_space {
                     Err(res) => outbox.push(res),
                     Ok(space_gateway) => {
-                        let response = FetchDhtEntryResponseData {
-                            msg_id: msg.request_id.clone(),
-                            entry: msg.entry.clone(),
-                        };
-                        let cmd = DhtCommand::EntryDataResponse(response);
-                        space_gateway.post_dht(cmd)?;
+                        if msg.request_id == "__author_list" {
```

Comment:
: ) - I think we might need to create a rust equivalent of https://github.com/holochain/n3h/blob/master/lib/n3h-common/track.js at some point.

Reply:
Yeah­. I'll add a todo for that.